includes/polling/applications/freeradius.inc.php

```diff
@@ -21,17 +21,45 @@
     $freeradius[$var] = $value;
 }
 
-//FreeRADIUS-Total-Access
+// Check $freeradius array for missing keys, returns null when the key does not exist.
+$get = function ($key) use ($freeradius) {
```

Comment:
most of this function already exists in Number::cast()

Reply:
Cheers, @murrant .  I level the key check and trims and change the number handling to Number::cast().